src/olympia/hero/tests/test_models.py

```diff
@@ -68,31 +75,28 @@ def test_clean_gradient_and_image(self):
         with self.assertRaises(ValidationError) as ve:
             ph.clean()
         assert 'gradient_color' in ve.exception.error_dict
-        assert 'image' not in ve.exception.error_dict
+        assert 'select_image' not in ve.exception.error_dict
 
-        ph.update(image='foo.png')
+        ph.update(select_image=self.phi)
         with self.assertRaises(ValidationError) as ve:
             ph.clean()
         assert 'gradient_color' in ve.exception.error_dict
-        assert 'image' not in ve.exception.error_dict
+        assert 'select_image' not in ve.exception.error_dict
 
-        ph.update(image='', gradient_color='#123456')
+        ph.update(select_image=None, gradient_color='#123456')
         ph.clean()  # it raises if there's an error
 
     def test_clean_only_enabled(self):
         hero = PrimaryHero.objects.create(
             promoted_addon=PromotedAddon.objects.create(
                 addon=addon_factory(), group_id=RECOMMENDED.id),
-            gradient_color='#C60184', image='foo.png')
+            gradient_color='#C60184', select_image=self.phi)
         PromotedApproval.objects.create(
             version=hero.promoted_addon.addon.current_version,
             group_id=RECOMMENDED.id)
         assert not hero.enabled
         assert not PrimaryHero.objects.filter(enabled=True).exists()
         # It should still validate even if there are no other enabled shelves,
-        # because we're not changing its enabled state.
-        hero.clean()  # it raises if there's an error
```

Comment:
does this fail now?

Reply:
This was accidentally removed, it has been added back in in the latest push and passes.